addons/mail/models/mail_followers.py

```diff
@@ -142,24 +142,23 @@ def _get_recipient_data(self, records, message_type, subtype_id, pids=None, cids
         LEFT JOIN res_users users ON users.partner_id = partner.id AND users.active
         LEFT JOIN res_groups_users_rel groups_rel ON groups_rel.uid = users.id
         LEFT JOIN res_groups groups ON groups.id = groups_rel.gid
-        WHERE EXISTS (
-            SELECT partner_id FROM sub_followers
-            WHERE sub_followers.channel_id IS NULL
-                AND sub_followers.partner_id = partner.id
-                AND (coalesce(sub_followers.internal, false) <> TRUE OR coalesce(partner.partner_share, false) <> TRUE)
-        ) %s
+        LEFT JOIN sub_followers subfol ON subfol.partner_id = partner.id
+        WHERE %s
+            subfol.channel_id IS NULL
+            AND subfol.partner_id = partner.id
```

Comment:
already checked in JOIN.

Reply:
It doesn't work without this condition https://runbot.odoo.com/runbot/build/5601769

Reply:
The condition can be deleted only with INNER JOIN